privacyidea/lib/eventhandler/base.py

```diff
@@ -444,6 +451,12 @@ def check_condition(self, options):
             if is_true(condition_value) != is_true(result_status):
                 return False
 
+        if CONDITION.RESULT_AUTHENTICATION in conditions:
+            condition_value = conditions.get(CONDITION.RESULT_AUTHENTICATION)
+            result_auth = content.get("result", {}).get("authentication")
+            if is_true(condition_value) != is_true(result_auth):
+                return False
```

Comment:
can you add a testcase to cover this condition?

Reply:
Oups, thank you for asking.
I think this is actually *wrong code*. Since the contents of result->authentication is not kind of true, but it is one of the four defined strings. So I actually have to fix the code.

Reply:
fixed it in 9c4e630f3